<ui version="4.0" >
 <class>RaporRolePerGorev</class>
 <widget class="QWidget" name="RaporRolePerGorev" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Rolelerdeki Personel Görevleri</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Ara :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="arama_kutusu" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QSplitter" name="splitter_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="tablo" >
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
      <widget class="QTableView" name="tablo_mevcut_gorevler" >
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="textEdit" />
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QPushButton" name="pBYazdir" >
     <property name="text" >
      <string>Çıktı Al</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
